src/apicompat/netstandard/netstandard.depproj

```diff
@@ -2,9 +2,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
-    <TargetFramework Condition="'$(TargetFramework)'==''">netstandard1.6</TargetFramework>
+    <TargetFramework Condition="'$(TargetFramework)'==''">netstandard2.0</TargetFramework>
     <OutputFolder>$(TargetFramework)</OutputFolder>
```

Comment:
Is `OutputFolder` still used for anything after this? Why get rid of it?

Reply:
I think it should be safe to delete it. I wasn't sure as I didn't know if its used by the depproj files.